### PR TITLE
TST: Fix 2.6 hfe4a6c

### PR DESCRIPTION
**Pull request by hsinfan1996**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
This is good for debugging. When we went to merge though we'll either need a patch or a new ccl release.

**Comment by hsinfan1996** [comment]
> This is good for debugging. When we went to merge though we'll either need a patch or a new ccl release.

Yes

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 36c2cdbbf4086cc196f7ee7d6e4bab7e190033ce9af409238cfc119ddafbd49a
-  patches:
-    - remove_setuptools_build.patch
```
Comment (beckermr) [inline recipe/meta.yaml:12]: You need to keep this patch and use the build.sh file for sure.

**Comment by beckermr** [comment]
What cmake changes did you need here?

**Comment by hsinfan1996** [comment]
> What cmake changes did you need here?

Nothing, changes will be made in the CCL repo.
https://github.com/LSSTDESC/CCL/compare/master...force_openmp_flag

**Comment by hsinfan1996** [comment]
Done testing.

**Pull request closed without merging**